app/views/proposals/index.html.haml

```diff
@@ -47,7 +47,7 @@
                         .proposal-meta.proposal-description
                           .proposal-meta-item
                             %strong Track:
-                            = invitation.proposal.track_name
+                            = invitation.proposal.track.name
```

Comment:
Actually you'll wanna use the proposal decorator's track_name method, which has a bit of extra logic that's intended for use in views. Have a look at that method to see what I'm talking about.